myscript.js

```diff
@@ -0,0 +1,21 @@
+
+var dataArray = [1, 8, 7, [ 4, 65, 1, [ 2, 7 ], 45, [ 6, 4], 6 ], 4, 8]
+
+var jenyaFlatternArray = function (Array) {
```

Comment:
Sure, it's my fault to name variable starting with upper case. I already know it but didn't pay enough attention. 
Also, can You recommend me where can I read more about ''traditional code writing rules" like using ' instead of ", spacing etc.